types/utils.go

```diff
@@ -40,39 +38,53 @@ func GetRandomValidatorSetWithPrivKey() (*cmtypes.ValidatorSet, ed25519.PrivKey)
 	return valset, privKey
 }
 
-// GetValidatorSet returns a validator set with a single validator
-// with the given key
-func GetValidatorSet(privKey ed25519.PrivKey) *cmtypes.ValidatorSet {
-	pubKey := privKey.PubKey()
-	valset := cmtypes.NewValidatorSet(
-		[]*cmtypes.Validator{
-			{PubKey: pubKey, Address: pubKey.Address(), VotingPower: 1},
-		},
-	)
-	return valset
+// BlockContext carries all necessary state for block generation, including configurations and outputs.
+type BlockContext struct {
```

Comment:
Context is a used term, as it relates to `context.Context`.
Let's call this `BlockConfig` or `BlockOptions`.